extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -378,6 +383,24 @@ export class AmpStoryPage extends AMP.BaseElement {
     }
   }
 
+  /** @private */
+  maybeSetStoryNextUp_() {
```

Comment:
nit: A comment here to give context on storyNextUp could be helpful for scanning the code.

Reply:
Done. Let me know if this looks good to you.